JoinColony/block-ingestor
# Implement Remove Verified Members

##Description

Implements removing verified members

##How to test

1. Stop the block ingestor while running npm run dev, and run this branch instead.
2. Spin up the CDapp on (tba)
3. Choose a verified member and copy their address
4. go to the colony dashboard home, where the temporary component for removing verified members lives, paste the address and click Unverify
5. Check logs to see that block-ingestor processed the transaction
6. Use GraphQL query getVerifiedMembersByColony to check the users has been removed. The user you have pasted in above should no longer show up in the items array

Resolves #161
